service/src/main/kotlin/fi/espoo/evaka/invoicing/domain/FeeThresholdsWithValidity.kt

```diff
@@ -144,6 +108,14 @@ data class FeeThresholds(
             4 -> maxIncomeThreshold2
             5 -> maxIncomeThreshold5
             6 -> maxIncomeThreshold6
-            else -> maxIncomeThreshold6 + ((familySize - 6) * thresholdIncrease6Plus)
+            else -> maxIncomeThreshold6 + ((familySize - 6) * incomeThresholdIncrease6Plus)
+        }
+
+    fun siblingDiscountPercent(siblingOrdinal: Int): Int =
```

Comment:
I guess this should be the coefficient not percentage

Reply:
I agree. Why not refactor it now?